release/pkg/generate_spec.go

```diff
@@ -146,6 +146,11 @@ func (r *ReleaseConfig) GetVersionsBundles(imageDigests map[string]string) ([]an
 		return nil, errors.Wrapf(err, "Error getting bundle for Bottlerocket admin container")
 	}
 
+	packageBundle, err := r.GetPackagesBundle(imageDigests)
```

Comment:
Can we add these under a dev-release check so they don't appear in a production release bundle? Refer to line 156 below.

Reply:
Fixed in https://github.com/aws/eks-anywhere/commit/ee9b00a4cb639d51f9815f1baf00c374f6d91473